qmf/src/plugins/messageservices/imap/imapauthenticator.cpp

```diff
@@ -127,7 +127,7 @@ static QByteArray authenticationResponses(QList<QByteArray> &authList, const QMa
                 authPassword = QString::fromLatin1(authList.takeFirst());
                 responseAuthType = QMail::CramMd5Mechanism;
             } else {
-                gResponses[id] = authList;
+                gResponses[id] = authList.takeFirst();
```

Comment:
responseAuthType remains unmodified, is that intentional?

Generally, global variables is a pretty fragile mechanism of passing parameters between functions. But that's a design issue, unrelated to this commit.